datafusion/common/src/scalar.rs

```diff
@@ -431,122 +387,6 @@ impl PartialOrd for ScalarValue {
     }
 }
 
-/// This function computes the duration (in milliseconds) of the given
```

Comment:
These functions are somewhat misleading as it is not clearly conveyed that they are a gross approximation, assuming certain numbers of days in months, etc... Given these don't appear to be used elsewhere, I think the best option is to remove them